packages/layout/src/ChartPanel.ts

```diff
@@ -7,6 +7,11 @@ export class ChartPanel extends Border2 implements IClickHandler {
 
     private _toggleLegend: ToggleButton = new ToggleButton(this, "fa-info").selected(false);
     private _buttonDownload: Button = new Button(this, "fa-download");
+    private _buttonClose: Button = new Button(this, "fa-close");
+    private _buttonMinimize: Button = new Button(this, "fa-minimize");
+    private _buttonMaximize: Button = new Button(this, "fa-maximize");
```

Comment:
I think these 3 buttons should be a single toggle button (when on, its maximized and when off its not)?  Or maybe a TriStateButton?